### PR TITLE
Replace deprecated functions with newer ones

### PR DESCRIPTION
**Pull request by Panquesito7**
This PR replaces deprecated functions with newer ones.
**However, it may not work (properly) with the 0.4-series.**

Tested with MT/MTG 5.0.1 and the mod works fine (found no errors/bugs).
If you find any bug/error/typo (on my PR), please let me know, we can work together to fix it.

If there's something you don't like, you can change it if you wish.
If this PR creates bugs/errors, feel free to close it.

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -32,7 +32,7 @@ if not minetest.registered_privileges[areas.config.self_protection_privilege] th
 	})
 end
 
-if minetest.settings:get_bool("log_mod") then
+if minetest.settings:get_bool("log_mods") then
```
Comment (SmallJoker) [inline init.lua:35]: Needs documentation (somewhen)

**Review by SmallJoker — APPROVED** [review]

File: init.lua (hunk 1 above)
Comment (Panquesito7) [reply init.lua:35]: Should I resolve this conversation, or do something else?

Comment (SmallJoker) [reply init.lua:35]: Was just a note. If you find more stuff to fix in areas, you could take the opportunity to document this setting. Minor issue anyway.

**Comment by Panquesito7** [comment]
On file "README.md" shouldn't 5.0.0+ requiered to run areas?

**Comment by Panquesito7** [comment]
@SmallJoker, thanks for merging the Pull Request.
Just one thing... On file `README.md` shouldn't 5.0.0+ be the requiered version to run areas (`Areas mod for Minetest 5.0.0+`)?
It says "Areas mod for Minetest 0.4.8+". That version is...kinda old.

**Comment by SmallJoker** [comment]
`set_pos` is 0.4.16+. Will adjust the readme accordingly.

**Comment by Panquesito7** [comment]
Sounds good to me. 👍

**Pull request merged**